charts/cluster-autoscaler/README.md

```diff
@@ -460,8 +460,9 @@ vpa:
 | secretKeyRefNameOverride | string | `""` | Overrides the name of the Secret to use when loading the secretKeyRef for AWS and Azure env variables |
 | securityContext | object | `{}` | [Security context for pod](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/) |
 | service.annotations | object | `{}` | Annotations to add to service |
+| service.clusterIP | string | `""` | IP address to assign to service |
 | service.create | bool | `true` | If `true`, a Service will be created. |
-| service.externalIPs | list | `[]` | List of IP addresses at which the service is available. Ref: https://kubernetes.io/docs/user-guide/services/#external-ips. |
+| service.externalIPs | list | `[]` | List of IP addresses at which the service is available. Ref: https://kubernetes.io/docs/concepts/services-networking/service/#external-ips. |
```

Comment:
Yes I know it's more work but let's drop these link fixes and submit a separate PR to address them.

Reply:
ok @jackfrancis, I will open a seperate PR for this one.

Hi @eshiettjoseph, please update the PR.

Reply:
I have raised a PR #7317 to fix the link.